### DISCUSSION TITLE
Redirection to Privatebin after Oauth2-proxy not working

### DISCUSSION THREAD

**Discussion by GopalGupta1805** [PrivateBin/PrivateBin#1186] [Q&A]
### Describe the problem/question

I have Privatebin setup in K8s cluster which has Oauth in-front of this. When opening a secret URL like `https://example.com/?<key>#<value>` after successful authentication, It lands on an empty page shared below instead of the same URL that was hit previously

![image](https://github.com/PrivateBin/PrivateBin/assets/60381500/3db5ed15-e80f-4c19-8100-4bd3f38ca9c7)

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. Setup Privatebin chart in GKE K8s cluster
2. Setup Oauth2-proxy chart in GKE K8s cluster in front of above chart
3. Setup nginx-ingress for loadbalancer in front for both
...

### What happens

**When opening a secret URL like `https://example.com/?<key>#<value>` after successful authentication, It lands on an empty page**
![image](https://github.com/PrivateBin/PrivateBin/assets/60381500/b2bafb08-7ed7-498b-94c2-9ed80b36a499)

### What should happen

Instead it should land on page which has secrets that is shared on it.

### Issue reproducibility

Yes, reproducible on https://privatebin.net.

**Comment by elrido** [comment]
You will have to review the exact requests and redirects issued by your setup, for example using your browsers F12 console using the network traffic tab. Very likely, the oauth server tries to redirect you back to the original page, but it won't have access to the hash-part of your URL (the bit after the "#" which is the secret key of the paste) and so the privatebin application will switch into the "new paste" mode.
